src/constructs/loadbalancing/alb/application-listener.ts

```diff
@@ -45,14 +45,17 @@ export class GuHttpsApplicationListener extends GuAppAwareConstruct(ApplicationL
     const { certificate, targetGroup } = props;
 
     const mergedProps: GuApplicationListenerProps = {
-      port: 443,
-      protocol: ApplicationProtocol.HTTPS,
+      port: typeof certificate === "undefined" ? 8080 : 443,
```

Comment:
I think we can be a bit terser here, removing the `typeof`:

```ts
const port = certificate ? 443 : 8080;
```

Same applies for `protocol` and `certificates` below.